pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -767,6 +769,44 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 		})
 	})
 
+	It("when network VMI spec validator execute, should", func() {
+		controller.validateNetworkSpec = validateNetVMISpecStub()
+
+		vmi := newPendingVirtualMachine("testvmi")
+		addVirtualMachine(vmi)
+
+		sanityExecute()
+
+		testutils.ExpectEvent(recorder, kvcontroller.SuccessfulCreatePodReason)
+
+		vmi, err := virtClientset.KubevirtV1().VirtualMachineInstances(vmi.Namespace).Get(context.Background(), vmi.Name, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		Expect(vmi.Status.Conditions).To(HaveLen(1))
+		Expect(vmi.Status.Conditions).To(SatisfyAll(ContainElement(MatchFields(IgnoreExtras, Fields{
+			"Type":   Equal(virtv1.VirtualMachineInstanceReady),
+			"Status": Equal(k8sv1.ConditionFalse),
+			"Reason": Equal(virtv1.GuestNotRunningReason),
+		}))))
```

Comment:
Is there an alternative way to make sure that function was called with less assertions?

Reply:
I think the positive scenario (when the network spec validator succeeds) is implicitly tested on multiple tests.
Please consider dropping this test.

Reply:
Done